lightningd/channel/channel.c

```diff
@@ -271,6 +263,9 @@ static void init_channel(struct peer *peer, const u8 *msg)
 				    &points[LOCAL], &points[REMOTE],
 				    am_funder ? LOCAL : REMOTE);
 
+	peer->channel_direction =
+	    pubkey_cmp(&peer->node_ids[LOCAL], &peer->node_ids[REMOTE]) > 0;
+
```

Comment:
I guess inlining in this case means creating a `#define` that'd replace the call with the method body, right?